boxsdk/client/client.py

```diff
@@ -317,6 +317,39 @@ def create_group(self, name):
         )
 
     @api_call
+    def get_recent_items(self, limit=100, offset=0, fields=None):
+        """
+        Get the user's recently accessed items.
+
+        :param: limit
+            The maximum number of items to return
+        :type: limit
+            `int`
+        :param offset:
+            The index at which to start returning items.
+        :type offset:
+            `int`
+        :param fields:
+            List of fields to request.
```

Comment:
Mention that the fields apply to the recent item object, not to the underlying file or folder.